README.md

```diff
@@ -407,6 +407,19 @@ These settings can be changed at any time.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
 * `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
 
+#### Image registry settings
+
+The following setting is optional and allows you to configure image registry mirrors and pull-through caches for your orchestrated containers.
```

Comment:
Does this only cover orchestrated containers and not host containers?  Should it cover both?

Reply:
Right, this currently only covers orchestrated containers (containers via cri-containerd). 

For host-containers it's not immediately clear to me how I can configure the containerd client to use registry mirrors. I suspect I'd have to modify resolver to accomplish that? I can create a separate issue to track registry mirrors for host-containers.

Reply:
Yes, this would be something that would have to be done in `host-ctr`.  I'm not sure if we should have the setting called `settings.container-registry.mirrors` if it only controls orchestrated containers though; how would we change this to handle host containers in the future?

Reply:
It seems like it's just a bug that `host-ctr` and host containers wouldn't support the mirror settings - same as if a component doesn't support the system-wide proxy settings.